Packs/SafeBreach/Playbooks/SafeBreach_Process_Non-Behavioral_Insights_Feed.yml

```diff
@@ -0,0 +1,460 @@
+      name: SafeBreach - Rerun Insights
+      description: 'This playbook reruns a SafeBreach insight based on id and waits
+        until it completes. '
+      playbookName: SafeBreach - Rerun Insights
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "18"
+    scriptarguments:
+      InsightIds:
+        complex:
+          root: InsightIds
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 490,
+          "y": 480
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "20":
+    id: "20"
+    taskid: a6d2d12a-5d5d-468f-8472-5c934b07c6f8
+    type: regular
+    task:
+      id: a6d2d12a-5d5d-468f-8472-5c934b07c6f8
+      version: -1
+      name: Get Insight Info
+      description: Gets SafeBreach Insights for all security control categories.
+      script: '|||safebreach-get-insights'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "21"
+    scriptarguments:
+      insightIds:
+        complex:
+          root: InsightIds
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 490,
+          "y": 130
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "21":
+    id: "21"
+    taskid: 4f69b1bd-18c9-4fe7-83e7-9f39ded9cb91
+    type: regular
+    task:
+      id: 4f69b1bd-18c9-4fe7-83e7-9f39ded9cb91
+      version: -1
+      name: Get Insight Remediation Data
+      description: Gets remediation data for a specific SafeBreach Insight.
+      script: '|||safebreach-get-remediation-data'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      insightId:
+        complex:
+          root: InsightIds
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 490,
+          "y": 300
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 1665,
+        "width": 380,
+        "x": 490,
+        "y": -680
+      }
+    }
+  }
+inputs:
+- key: ""
+  value: {}
+  required: false
+  description: ""
+  playbookInputQuery:
+    query: sourceBrands:["SafeBreach*"] and -safebreachisbehavioral:T
+    queryEntity: indicators
+    results:
+    daterange:
+      fromdate: 0001-01-01T00:00:00Z
+      todate: 0001-01-01T00:00:00Z
+      period:
+        by: ""
+        byto: ""
+        byfrom: ""
+        tovalue:
+        fromvalue:
+        field: ""
+      fromdatelicenseval: 0001-01-01T00:00:00Z
+    runFromLastJobTime: true
+outputs: []
```

Comment:
Shouldn't the playbook have outputs or everything is done just in the sublaybooks?

Reply:
it is the main processing playbook it performs all the tasks within it there is no expectation to use this playbook as sub playbook.